packages/seo/metadata.ts

```diff
@@ -14,6 +14,7 @@ const author: Metadata['authors'] = {
 };
 const publisher = 'Hayden Bleasel';
 const twitterHandle = '@haydenbleasel';
+const siteUrl = 'https://www.next-forge.com/';
```

Comment:
2 issues here:

1. This should be `env.NEXT_PUBLIC_VERCEL_PROJECT_PRODUCTION_URL` which represents the production URL of this project.
2. This is the default value anyway: https://nextjs.org/docs/app/api-reference/functions/generate-metadata#default-value